addons/binding/org.openhab.binding.icloud/README.md

```diff
@@ -0,0 +1,136 @@
+If a device is removed from the account the respective openHAB thing will go OFFLINE.
+
+## Channels
+
+### Account
+
+| Channel ID      | Type   | Description                                                  |
+|-----------------|--------|--------------------------------------------------------------|
+| numberOfDevices | Number | Number of registered devices with this iCloud account.       |
+| owner           | String | Registered owner of this iCloud account.                     |
+| refresh         | Switch | Update all devices registered with this account immediately. |
+
+### Device
+
+The following channels are available (if supported by the device):
+
+| Channel ID                   | Type     | Description                                                                                                                                 |
+|------------------------------|----------|---------------------------------------------------------------------------------------------------------------------------------------------|
+| batteryStatus                | String   | Current battery status (Charging, Charged, Unknown)                                                                                         |
+| batteryLevel                 | Number   | Battery charge in %                                                                                                                         |
+| findMyPhone                  | Switch   | Triggers the ["Find my phone"](https://support.apple.com/explore/find-my-iphone-ipad-mac-watch) functionality of the device (if available). |
+| location                     | Location | Location of the device.                                                                                                                     |
+| locationAccuracy             | Number   | Accuracy of the last position report.                                                                                                       |
+| locationLastUpdate           | DateTime | Timestamp of the last location update.                                                                                                      |
+| distanceFromHome             | Number   | Device distance from the location set in openHAB regional settings.                                                                         |
+| addressStreet<sup>i</sup>    | String   | Street component of the address.                                                                                                            |
+| addressCity<sup>i</sup>      | String   | City component of the address.                                                                                                              |
+| addressCountry<sup>i</sup>   | String   | Country component of the addess.                                                                                                            |
```

Comment:
I remember seeing more than these three properties in the google geocode API. There was e.g. mentioning of the neighborhood of a city, which would be interesting. Please consider adding them as channels.

Reply:
For the 1st version of the binding I will keep the currently supported channels; once the code is backmerged we can have a look what other channels might be useful to users.

Reply:
Update: As this feature should not be part of the binding I´ve removed the address lookup functionality completely.